supplementary/alica_ros1/alica_ros_proxy/include/clock/AlicaRosTimer.h

```diff
@@ -23,7 +23,7 @@ class SyncStopTimerRosImpl : public std::enable_shared_from_this<SyncStopTimerRo
 public:
     using TimerCb = std::function<void()>;
 
-    SyncStopTimerRosImpl(CallbackQ& cbQ, TimerCb&& userCb, alica::AlicaTime period)
+    SyncStopTimerRosImpl(CallbackQ& cbQ, TimerCb&& userCb, const alica::AlicaTime& period)
```

Comment:
Seems small enough to pass by value.

Reply:
Generally dangerous assumption which relies on current and internal object implementation, especially 3rd party.